src/main/java/Dog.java

```diff
@@ -34,4 +36,11 @@ public void eat(){
             this.eat=true;
         }
     }
+
+    public void walk(){
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Rename method "walk" to prevent any misunderstanding/clash with field "walk" defined on line 8. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1845)